spec/system/tasks_spec.rb

```diff
@@ -0,0 +1,70 @@
+require 'rails_helper'
+
+RSpec.describe "Tasks", type: :system do
+
+  describe 'ログイン前' do
+    context 'タスク作成するとき' do
+      it 'タスク作成が失敗' do
+      end
+    end
+    context 'タスク編集するとき' do
+      it 'タスク編集が失敗' do
+      end
```

Comment:
ログインしないと画面に遷移できない場合は、以下のような形で良いかもしれません。
ログインしていない状態で、作成処理や編集処理を厳密に行わせたい場合は「特定のリクエストをリクエストスペックで送る」などの記載が必要になるかもしれませんが、SystemSpecではそこまで確認しなくて大丈夫です。

APIモードでRailsの機能を作成した場合は、上記の確認を行っても良いかもしれません。
```suggestion
    context 'タスク作成画面に遷移' do
      it 'ログイン画面に遷移すること' do
      end
    end
    context 'タスク編集画面に遷移' do
      it 'ログイン画面に遷移すること' do
      end
```